js/src/scrollspy.js

```diff
@@ -133,7 +133,7 @@ const ScrollSpy = (($) => {
             target = $(targetSelector)[0]
           }
 
-          if (target && (target.offsetWidth || target.offsetHeight)) {
+          if (target && (target.getBoundingClientRect().width || target.getBoundingClientRect().height)) {
```

Comment:
Yes this is what I did at the beginning, but I have to check if target is non null, so the code would have been:
```js
if (target) {
  const targetBCR = target.getBoundingClientRect()
  if (target.targetBCR.width || targetBCR.height) {
   //Do something
  }
}
```
So that makes longer code and less legible IMO. That said, I don't know if the browser has to compute the result for each call or if it just return a result that was already there.
I'll do some research quickly to check and I'll change the code accordingly.